src/api/ImportHelpers.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace BDMS;
+
+/// <summary>
+/// Helper methods for importing data.
+/// </summary>
+public static class ImportHelpers
```

Comment:
REQ: Dieser Titel passt nicht (mehr?) zum Inhalt der Klasse. Die Klasse beinhaltet lediglich eine Methode für den Vergleich von zwei `double`-Werten mit Toleranz. Entweder Klassennamen anpassen oder die Methode in eine bereits existierende Klasse auslagern.

Reply:
Absolut 👍🏻 Da war einmal viel mehr drin bis praktisch alles wieder rausgeflogen ist, weils nicht gepasst hat. Habs ein bisschen umstrukturiert.